libs/cardano-ledger-core/CHANGELOG.md

```diff
@@ -6,6 +6,19 @@
 * Remove unnecessary instances: `Inject a ()` and `Inject Void a`
 * Add `integralToBounded` to `BaseTypes`
```

Comment:
Additional suggestions for the changelog:
 * add `PlutusScriptContext` type family
 * add `LegacyPlutusArgs` data 
 * add  `PlutusArgs` data family to `PlutusLanguage` and instances
 * change signature of `evaluatePlutusRunnable` and `evaluatePlutusRunnableBudget` to replace data list with `PlutusArgs`